monkvision/monkjs
# Enhance corejs Feat create one inspection

Proper use of the staging API while creating a new inspection when we mount CameraPreview screen.
> Reminder: in order to save images, we need to create a "blank" inspection and use its id

How to test:
* Run project on web browser
* Open console
* Start capture process
* See console and networks logs
